### PR TITLE
Added missing `items` auto completion

### LINKED ISSUE #198: Items are not auto completed in parameter definition
Auto completion does not work in `items` in the following case:

```yaml
swagger: '2.0'
parameters:
  Dog:
    items:
      <caret>
```

### PR DESCRIPTION
**Pull request by semonte**
This commit adds auto completion in the following case:

```yaml
swagger: '2.0'
parameters:
  Dog:
    items:
      <caret>
```

Fixes #198

### CODE REVIEW

File: src/test/java/org/zalando/intellij/swagger/completion/field/completion/swagger/SwaggerCompletionTest.java [hunk 1]
```diff
@@ -178,6 +178,15 @@ public void thatItemsKeysInDefinitionsAreSuggested() {
                 .isOfSize(30);
     }
 
+    @Test
+    public void thatItemsKeysInParameterDefinitionsAreSuggested() {
+        getCaretCompletions("items_in_parameter_definitions")
+                .assertContains("type", "format", "items", "collectionFormat", "default", "maximum",
+                        "exclusiveMaximum", "minimum", "exclusiveMinimum", "maxLength", "minLength", "pattern",
+                        "maxItems", "minItems", "uniqueItems", "multipleOf")
+                .isOfSize(17);
```
Comment (maxim-tschumak) [inline src/test/java/org/zalando/intellij/swagger/completion/field/completion/swagger/SwaggerCompletionTest.java:187]: There are 16 completions items that have to be present and a check for the total number (17). Maybe including the 17th item and removing the check for the total number would make the test more robust.

**Review by maxim-tschumak — APPROVED** [review]

**Comment by maxim-tschumak** [comment]
:+1:

File: src/test/java/org/zalando/intellij/swagger/completion/field/completion/swagger/SwaggerCompletionTest.java (hunk 1 above)
Comment (semonte) [reply src/test/java/org/zalando/intellij/swagger/completion/field/completion/swagger/SwaggerCompletionTest.java:187]: Nice catch, fixed!

**Comment by maxim-tschumak** [comment]
:+1:

**Pull request merged**